build/lex.go

```diff
@@ -642,6 +650,8 @@ func (in *input) order(v Expr) {
 		// nothing
 	case *Ident:
 		// nothing
+	case *BranchStmt:
```

Comment:
future clean-up: can use , to combine 'nothing' cases

case *Ident, *BranchStmt, *StringExpr:
   // nothing